x-pack/plugins/integration_assistant/server/graphs/ecs/constants.ts

```diff
@@ -1703,92 +1751,314 @@ export const ECS_FIELDS: EcsFields = {
+  'email.cc.domain': 'Domain of CC recipients.',
+  'email.cc.local': 'Local part of the CC recipients.',
+  'email.cc.registered_domain': 'The highest registered CC domain, stripped of the subdomain.',
+  'email.cc.subdomain': 'Subdomain of the CC recipients.',
+  'email.cc.top_level_domain': 'Top level domain of the CC recipients.',
+  'email.content_type': 'Content type of the email.',
+  'email.delivery_timestamp': 'Time the email was delivered.',
+  'email.direction': 'Direction of the email.',
+  'email.from.address': 'Email address of the sender.',
+  'email.from.domain': 'Domain of the sender.',
+  'email.from.local': 'Local part of the sender.',
+  'email.from.registered_domain':
+    'The highest registered sender domain, stripped of the subdomain.',
+  'email.from.subdomain': 'Subdomain of the sender.',
+  'email.from.top_level_domain': 'Top level domain of the sender.',
+  'email.local_id': 'Local identifier of the email.',
+  'email.message_id': 'Message ID of the email.',
+  'email.origination_timestamp': 'Time the email was originated.',
+  'email.reply_to.address': 'Email address of the reply-to.',
+  'email.reply_to.domain': 'Domain of the reply-to.',
+  'email.reply_to.local': 'Local part of the reply-to.',
+  'email.reply_to.registered_domain':
+    'The highest registered reply-to domain, stripped of the subdomain.',
+  'email.reply_to.subdomain': 'Subdomain of the reply-to.',
+  'email.reply_to.top_level_domain': 'Top level domain of the reply-to.',
+  'email.sender.address': 'Email address of the sender.',
+  'email.sender.domain': 'Domain of the sender.',
+  'email.sender.local': 'Local part of the sender.',
+  'email.sender.registered_domain':
+    'The highest registered sender domain, stripped of the subdomain.',
+  'email.sender.subdomain': 'Subdomain of the sender.',
+  'email.sender.top_level_domain': 'Top level domain of the sender.',
+  'email.size': 'Size of the email in bytes.',
+  'email.subject': 'Subject of the email.',
+  'email.to.address': 'Email addresses of recipients.',
+  'email.to.domain': 'Domain of recipients.',
+  'email.to.local': 'Local part of the recipients.',
+  'email.to.registered_domain':
+    'The highest registered recipient domain, stripped of the subdomain.',
+  'email.to.subdomain': 'Subdomain of the recipients.',
+  'email.to.top_level_domain': 'Top level domain of the recipients.',
+  'error.code': 'Error code describing the error.',
+  'error.id': 'Unique identifier for the error.',
+  'error.message': 'Error message.',
+  'error.stack_trace': 'Error stack trace.',
+  'error.type': 'Error type.',
   'event.action': 'The action captured by the event.',
-  'event.created': 'Time when the event was first read by an agent or by your pipeline.',
+  'event.category': 'Event category.',
   'event.code': 'Identification code for this event.',
+  'event.created': 'Date/time when the event was first read by an agent, or by your pipeline.',
+  'event.dataset': 'Name of the dataset.',
```

Comment:
do we want to omit this as well since we set it in the fields file : https://github.com/elastic/kibana/blob/main/x-pack/plugins/integration_assistant/server/templates/base_fields.yml.njk#L14 ?